api/spec/requests/contacts_controller_spec.rb

```diff
@@ -19,25 +19,31 @@
     } }
   end
 
-  describe "sends the message to installation" do
-    context "when params are valid" do
-      describe "the response" do
-        it "has a 204 status code" do
-          post api_v1_contacts_path, headers: headers, params: json_payload(valid_params)
-          expect(response).to have_http_status(204)
-        end
+  path '/contacts' do
+    post I18n.t('swagger.post.description', type: 'contact', attribute: 'ID') do
+      consumes 'application/json'
+      parameter name: :contact_params, :in => :body, schema: { '$ref' => '#/definitions/ContactRequestCreate' }
+      tags 'Contacts'
+
+      response '204', I18n.t('swagger.post.description', type: 'contact', attributes: 'ID') do
+        let(:contact_params) {{ data: valid_params }}
+        run_test!
+      end
+
+      response '422', I18n.t('swagger.unprocessable') do
+        let(:contact_params) {{ data: invalid_params }}
+        schema '$ref' => '#/definitions/ContactResponseError'
+        run_test!
       end
     end
+  end
 
+  describe "sends the message to installation" do
```

Comment:
This language is weird (not your fault, you didn't write it). Let's change it to "creates a new contact email"